language/move-vm/runtime/src/runtime.rs

```diff
@@ -29,6 +29,10 @@ impl VMRuntime {
         }
     }
 
+    pub(crate) fn loader(&self) -> &Loader {
```

Comment:
wasn't this supposed to go?

Reply:
It's used to create a `Session`, but yeah I can change it to a `pub(crate) fn new_session` so to not expose the `Loader`.